packages/deploy/deploy/800_oft_sand/803_oft_sand_setup_base.ts

```diff
@@ -18,17 +19,16 @@ const func: DeployFunction = async function (
     eidEthereum = process.env[`EID_${'SEPOLIA'}`];
     eidBsc = process.env[`EID_${'BSCTESTNET'}`];
   } else {
-    eidEthereum = 0;
-    eidBsc = 0;
+    throw new Error('Cannot find EID for network');
   }
 
-  const hreEthereum = hre.companionNetworks.mainnet;
+  const hreEthereum = hre.companionNetworks[DEPLOY_NETWORKS.ETH_MAINNET];
   const deploymentsEthereum = hreEthereum.deployments;
   const OFTAdapterForSand = await deploymentsEthereum.getOrNull(
     'OFTAdapterForSand'
   );
 
-  const hreBsc = hre.companionNetworks.bscMainnet;
+  const hreBsc = hre.companionNetworks[DEPLOY_NETWORKS.BSC_MAINNET];
   const deploymentsBsc = hreBsc.deployments;
   const OFTSandBsc = await deploymentsBsc.getOrNull('OFTSand');
 
```

Comment:
replace `OFTAdapterForSand` with `OFTSand` in isPeerForEthereum and isPeerForBsc